netlink/link.go

```diff
@@ -391,7 +391,7 @@ func SetLinkHairpin(bridgeName string, on bool) error {
 }
 
 // AddOrRemoveStaticArp sets/removes static arp entry based on mode
-func AddOrRemoveStaticArp(mode int, name string, ipaddr net.IP, mac net.HardwareAddr) error {
+func AddOrRemoveStaticArp(mode int, name string, ipaddr net.IP, mac net.HardwareAddr, isProxy bool) error {
```

Comment:
can you give a quick summary what the NTF flag is for?

Reply:
sure..its for neighbor proxy..not using right now